packages/react-server-test-pages/pages/bottleneck/elements.js

```diff
@@ -0,0 +1,33 @@
+import "./colors/red.scss";
+import "./colors/blue.scss";
+import "./colors/purple.scss";
+import "./colors/green.scss";
+import "./colors/orange.scss";
+import "./colors/indigo.scss";
+import "./colors/yellow.scss";
+
+const RedThing = () => <div className="red-thing">This should be red</div>;
+const BlueThing = () => <div className="blue-thing">This should be blue</div>;
+const PurpleThing = () => <div className="purple-thing">This should be purple</div>;
+const GreenThing = () => <div className="green-thing">This should be green</div>;
+const OrangeThing = () => <div className="orange-thing">This should be orange</div>;
+const IndigoThing = () => <div className="indigo-thing">This should be indigo</div>;
+const YellowThing = () => <div className="yellow-thing">This should be yellow</div>;
+const ColorWheel = [RedThing(), BlueThing(), PurpleThing(), GreenThing(),
+	OrangeThing(), IndigoThing(), YellowThing()];
+
+export default class ElementsPage {
+
+	getElements() {
+
+		const colorThings = [];
+		for (var i = 0; i < 99999; i++) {
+			let selection = Math.floor(Math.random() * 7);
```

Comment:
I think @roblg might be suggesting a replacement for the hard-coded 7 here.

I'll add that this can be a `const`. 👹